.gitallowed

```diff
@@ -7,11 +7,13 @@
 657299777109-kvb5qafr70bl01i6bnpgsiq5nt6v1o8u.apps.googleusercontent.com
 56507752110-ovdus1lkreopsfhlovejvfgmsosveda6.apps.googleusercontent.com
 684273740878-d7i68in5d9hqr6n9mfvrdh53snekp79f.apps.googleusercontent.com
+589109405884-bmoj9ra8849rqeepuamk8jpu102iq363.apps.googleusercontent.com
 63939010390-aj0r8hro7r8lkt7a45gissu3m73ietl2.apps.googleusercontent.com
 
 # Public API keys for Stackdriver. These also must be sent client-side.
 AIzaSyBSPcQ0FcR99GLXaqX4Ujpoj3JUDSP689g
 AIzaSyA4gOEvyJRkhIbW0x0Y7PkIowOSIK_Tous
 AIzaSyB44pugxPbfevF_fb7GichIv9D9ee5-nNk
 AIzaSyAkMIMIzUwv02RBK-A7cE1PbPpDJ2MTNtk
+AIzaSyA5Q0xqZdyaHUIUjZFqpim26wQFO0JqIRw
```

Comment:
Can we not just have a `.stackdriverKeys/*` or something for this? I realize they're public, but it still feels funny having these checked in.

Reply:
I don't understand the suggestion. Please review the purpose of this specially named file: https://github.com/awslabs/git-secrets#ignoring-false-positives

Reply:
Oh, OK. I thought they were file paths.